tests/test_render.py

```diff
@@ -1918,17 +1918,25 @@ def test_request_hooks(self):
                     {%scripts%}
                     <script id="_dash-renderer" type"application/json">
                         const renderer = new DashRenderer({
-                            request_pre: () => {
+                            request_pre: (payload) => {
                                 var output = document.getElementById('output-pre')
+                                var outputPayload = document.getElementById('output-pre-payload')
                                 if(output) {
                                     output.innerHTML = 'request_pre changed this text!';
                                 }
+                                if(outputPayload) {
+                                    outputPayload.innerHTML = JSON.stringify(payload);
+                                }
                             },
-                            request_post: () => {
+                            request_post: (payload) => {
```

Comment:
`request_post` also receives the response - we should test something about at least the basic structure of this response.